tools/sidekick/app.js

```diff
@@ -49,7 +49,7 @@
      */
     _initConfig(cfg = {}) {
       const prefix = (cfg && cfg.owner && cfg.repo)
-        ? `${cfg.ref ? `${cfg.ref}--` : ''}${cfg.repo}--${cfg.owner}`
+        ? `${cfg.ref && cfg.ref !== 'master' ? `${cfg.ref}--` : ''}${cfg.repo}--${cfg.owner}`
```

Comment:
humm... we are going away from `master`. And you cannot be sure if default is `main` or `master`... Is it really necessary ?

Reply:
Well, yes, because in both cases, it would not be part of the URL

Reply:
I'll include `main` for now

Reply:
Ok, cool. But I could be picky saying there is no test for this specific use case ;)